geonotebook/kernel.py

```diff
@@ -308,8 +314,10 @@ def __init__(self, kernel, *args, **kwargs):
         self.y = None
         self.z = None
         self.layers = GeonotebookLayerCollection([])
+        self.servers = { } # Dictionary servers.
 
         self._kernel = kernel
+        self._server_state = { }
```

Comment:
Initially the vis server was intended to be an ephemeral object that implemented an API that coordinated information between the tornado process and the kernel process. The coordinating state was intended to be statically defined in the config file. 

In general, I'm hesitant to introduce state now unless there is an immediate and compelling need. I'm also specifically concerned about maintaining the vis_server's state on a different object (e.g. the ```Geonotebook``` object).  If there is a compelling need to maintain state then i'd like to see if we can think of a better solution. 

Again,  i may be mis-reading this, but it seems like the immediate need is motivated by the introduction of the ```disgorge``` and the need to call ```unbind``` on the tms object passed into TMSRasterData?